packages/helpers/classes/attachment.js

```diff
@@ -39,17 +40,29 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId} = data;
+    const {content, filename, type, disposition, contentId, file} = data;
+
+    let fileContent = !file ? content : this.readFile(file);
 
     //Set data
-    this.setContent(content);
+    this.setContent(fileContent);
     this.setFilename(filename);
     this.setType(type);
     this.setDisposition(disposition);
     this.setContentId(contentId);
   }
 
   /**
+   * Read a file and return its content as base64
+   */
+  readFile(file) {
```

Comment:
If we use a separate setting for the encoding you can pass it in here, with a default value of `utf8`, e.g.

```
readFile(path, encoding = 'utf8') {
   ...
}
```